src/Navbar.js

```diff
@@ -123,8 +123,8 @@ const defaultProps = {
 };
 
 class Navbar extends React.Component {
-  constructor(...args) {
-    super(...args);
+  constructor(props) {
```

Comment:
changed, so that we have a consistency in the project.

Reply:
why not make `state` an instance property here as well?

Reply:
because then the context of it is incorrect. So I have to create a wrap function:
`onToggle: () => this.handleToggle(),`